include/coap3/coap_io_internal.h

```diff
@@ -147,12 +147,12 @@ void coap_epoll_ctl_add(coap_socket_t *sock, uint32_t events, const char *func);
 void coap_epoll_ctl_mod(coap_socket_t *sock, uint32_t events, const char *func);
 
 /**
- * Update the epoll timer fd as to when it is to trigger.
+ * Re-schedule when a call to coap_io_prepare_io is due.
  *
- * @param context The context to update the epoll timer on.
- * @param delay The time to delay before the epoll timer fires.
+ * @param context The context that requires the call.
+ * @param delay The time span until the call is due.
  */
-void coap_update_epoll_timer(coap_context_t *context, coap_tick_t delay);
+void coap_update_timer(coap_context_t *context, coap_tick_t delay);
```

Comment:
Perhaps a better function name is coap_update_refresh_timer() to give a clue to the reader as to what is happening here.  I do like the re-use of the coap_update_epoll_timer() concept to trigger the refresh of the libcoap logic when, say, a packet need to be re-transmitted.